### PR TITLE
Shrestha/tag rc2

### PR DESCRIPTION
**Pull request by shresthamalik**

### CODE REVIEW

File: ngraph_bridge/ngraph_executor.cc [hunk 1]
```diff
@@ -308,7 +308,8 @@ NGraphExecutor::CreateCallback(const std::string signature,
         "tf_function_" + m_node_name + "_" + to_string(rank_id) + ".json",
         serialized_ng_func);
     if (status != Status::OK()) {
-      return std::make_pair(status, std::make_pair(ng_exec, ng_function));
+      return std::make_pair(status,
```
Comment (kanvi-nervana) [inline ngraph_bridge/ngraph_executor.cc]: Isn't this the distributed build fix change that went in already?

Comment (shresthamalik) [reply ngraph_bridge/ngraph_executor.cc]: that went in the master, this is on r0.21 to create a tag

Comment (kanvi-nervana) [reply ngraph_bridge/ngraph_executor.cc]: May be you want to cherry-pick it separately so that tagging can be a stand-alone PR

Comment (shresthamalik) [reply ngraph_bridge/ngraph_executor.cc]: Its fine. Not a big change

**Pull request closed without merging**